src/main/scala/de/uhh/lt/spark/corpus/DefaultSource.scala

```diff
@@ -0,0 +1,110 @@
+package de.uhh.lt.spark.corpus
```

Comment:
please remove 'spark' and rename to de.uhh.lt.corpus
the whole project is about spark and none of its components will be ever used outside the JST namespace